MobileFilter/sections/whitelist.txt

```diff
@@ -23,6 +23,10 @@
 @@||kocaelibarisgazetesi.com/images/banner/l3oo.jpg
 ! https://forum.adguard.com/index.php?threads/www-dailymotion-com.31223
 @@||dmcdn.net^$app=com.dailymotion.dailymotion
+! https://github.com/AdguardTeam/AdguardFilters/issues/36075
+@@||securepubads.g.doubleclick.net/tag/js/gpt.js^$domain=sinarharian.com.my
+@@||securepubads.g.doubleclick.net/gpt/pubads_impl_2019070801.js^$domain=sinarharian.com.my
```

Comment:
`pubads_impl_2019070801.js` number is not constant. Replace by `/pubads_impl_*.js`
`^` is unnecessary

Reply:
@Alex-302, replaced by `/pubads_impl_*.js` and removed the `^`